parsl/channels/oauth_ssh/oauth_ssh.py

```diff
@@ -0,0 +1,140 @@
+import getpass
+import logging
+import select
+import paramiko
+import socket
+
+from parsl.errors import OptionalModuleMissing
+from parsl.channels.ssh.ssh import SSHChannel
+
+try:
+    from oauth_ssh.ssh_service import SSHService
+    from oauth_ssh.oauth_ssh_token import find_access_token
+    _oauth_ssh_enabled = True
+except (ImportError, NameError, FileNotFoundError):
```

Comment:
What's the FileNotFoundError that might happen here? Is it something that isn't explicitly import related?

Reply:
I don't think that's necessary. I believe that's for when modules might be coming from local dirs.

Reply:
Fixed.